test_meteoswiss/plot_output

```diff
@@ -43,14 +43,33 @@ Mandatory arguments to long options are mandatory for short options too.
   -f, --force            invoke pyflexplot even if plots already exist
   -h, --help             display this help and exit
   -n, --dry-run          print commands without executing them
+  -W, --dry-wet          additonally plot dry and wet deposition separately
   -N, --ncpus=NCPUS      use NCPUS parallel threads for pyflexplot
   -O, --operational      use operational pyflexplot from osm
   -p, --preset=PRESET    use pyflexplot with preset PRESET
-  -w, --wait             wait for batch job completion, allows to list results.
+  -s, --steps            plot step values rather than accumulated values
+  -w, --wait             wait for batch job completion, allows to list results
 
 EOF
 } # function display_help
 
+function cleanup_plot_dir {
+    # Check if plots exist
+    if ls $1/*p[dn][fg] > /dev/null 2>&1 ; then
+        echo -n $tag: "Plot files already exist in $1"
+        if [[ -n $clean ]] ; then
+            echo ", deleting."
+            rm -rf $1/*{png,pdf}
+        elif [[ -z $force ]] ; then
+            echo ", use --force to overwrite."
+            exe_cmd="echo -e $tag: SKIP: Plots already present, skipping:\n\e[07;32m \e[0m  "
+            wait_opt=skipped
+        else
+            echo ", overwriting."
+        fi
+    fi
+} # function cleanup_plot_dir
```

Comment:
```suggestion
function cleanup_plot_dir {

    # return early if no .png or .pdf files
    [[ -e $1/*.png || -e $1/*.pdf ]] || return

    echo -n "$tag: Plot files already exist in $d"
    if [[ -n $clean ]]; then
        echo ", deleting."
        rm -f -- $1/*{png,pdf}
    elif [[ -z $force ]]; then
        echo ", use --force to overwrite."
        exe_cmd=$'echo -e '"$tag: SKIP: Plots already present, skipping:\n\e[07;32m \e[0m  "
        wait_opt=skipped
    else
        echo ", overwriting."
    fi
}  # function cleanup_plot_dir
```

1. Consider using an early return instead of nested if statements.
2. The `-r` is probably not needed. Use `--` to prevent the glob from being treated as options if someone names a file -foo.pdf

Reply:
A nice suggestion - in principle. However the [[ -e *.pdf ]] does unfortunately not work if there is more than one file, which is usually the case here. So I keep my version with the `ls`. I know it's a bit hacky to use *p[dn][fg] rather than *.pdf and *.png, but I see now easy way to combine these into one `ls` command (`ls *.{pdf,png}` would fail even if only one type of file is missing)